test/schema/fixture/README.md

```diff
@@ -0,0 +1 @@
+# test
```

Comment:
There's appears to be a slight difference in how output is shown with Jest vs Mocha. But compiling the jsii fixture threw a warning that a readme was missing. This fixes that.